scripts/tests/misc.sh

```diff
@@ -13,6 +13,11 @@ test_package_with_feature primitives std
 
 for package in zrml/*
 do
+  if [[ "$package" == "zrml/market-commons" ]]; then
+    test_package_with_feature "$package" std
+    continue
+  fi
+
```

Comment:
There might be multiple pallets in the future that don't have the runtime-benchmarks feature, but I think it is reasonable to fix it when the second pallet without that feature is included.